src/server/database/migrations/20180423233147_create-projects_users.js

```diff
@@ -0,0 +1,23 @@
+export const up = async knex => {
+  await knex.schema.createTable('projects_users', table => {

```

Comment:
should we name the database `projectsUsers`?

Reply:
@PauuloG what's your opinion on this? I think we should go full camelCase 👍